pkg/statistics/handle/globalstats/global_stats_test.go

```diff
@@ -876,6 +876,7 @@ func TestGlobalIndexStatistics(t *testing.T) {
 		require.Nil(t, h.HandleDDLEvent(<-h.DDLEventCh()))
 		tk.MustExec("insert into t(a,b) values (1,1), (2,2), (3,3), (15,15), (25,25), (35,35)")
 		tk.MustExec("ALTER TABLE t ADD UNIQUE INDEX idx(b) GLOBAL")
+		<-h.DDLEventCh()
```

Comment:
What happens if we remove this?

Reply:
Because there's extra CreateIndex events, below logic like line 891 will pick wrong event to handle

Reply:
Got it.